tests/reference/runtime-test_list_02-5f7db5f.stderr

```diff
@@ -0,0 +1,2 @@
+IndexError: List index is out of range.
+Allocated index: 0:2, but given: 3
```

Comment:
Does the error message look good or understandable?
Do you have other suggestions?
@certik @czgdp1807

Reply:
For me following is better,

`IndexError: List index is out of range. Index range is (0, 2) but given index is 3.`

Reply:
Done!